src/Akka.Persistence.Azure.Hosting/AzurePersistenceExtensions.cs

```diff
@@ -12,65 +15,318 @@ namespace Akka.Persistence.Azure.Hosting
     /// </summary>
     public static class AzurePersistenceExtensions
     {
-        public const string DefaultTableName = "AkkaPersistenceDefaultTable";
```

Comment:
We should leave these public - and are these the same default values that are also in HOCON?

Reply:
Yes, those are the same default values in HOCON